multi_layer_network/src/event_baseline2.py

```diff
@@ -37,6 +37,8 @@ def event_baseline_linking(events, entity2cluster):
             id2 = IDs[j]
             if events[id1]['type'] == events[id2]['type']:
                 common = 0
+#                 What is this ? :
+#                             set_1.add(entity2cluster[ent])
```

Comment:
was in py3 branch with wired indent

Reply:
This code is from Xin's initial code. He must forget to remove them.